### DISCUSSION TITLE
ekşi sözlük deprem ihtiyaç özetleri

### DISCUSSION THREAD

**Discussion by ahmetcumhurarslan** [acikkaynak/deprem-yardim-projesi#52] [Ideas]
deprem bölgesindeki acil ihtiyaçların belirlenip sınıflandırmasını kolaylaştırmak için openai tabanlı özetleme scripti yazdım. bilgiler ekşi sözlükteki deprem yardımlaşma başlığından çekilmektedir ancak istenildiği takdirde başka kaynaklar da kullanılabilir.

çıktı olarak ihtiyaçların listesi, adres, telefon, isim gibi bilgileri bir tablo olarak vermektedir. Benzeri şeylerin twitter için kullanıldığını görüyorum. ekşi sözlük için yapılmış bir şey var mı bilmiyorum ama twitter dışındaki mecralar için de benzer şeylerin yapılmasında fayda olduğunu düşünüyorum (malum ksıtlamalar her an olabilir). bu platformda geliştirilen uygulamalara pek hakim değilim ne yazık ki, entegre edilebilecek bir bölüm olursa diye buraya yazıyorum. 

https://github.com/ahmetcumhurarslan/deprem-ihtiyac-ozeti

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz ve projeniz için teşekkürler. Ekşi Sözlük ile işbirliği çalışmamız bulunmaktadır.
